src/main/java/nl/novi/techiteasycontroller/exceptions/RecordNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package nl.novi.techiteasycontroller.exceptions;
+
+public class RecordNotFoundException extends RuntimeException {
+
+    public RecordNotFoundException(){
+        super("Television id not found");
```

Comment:
de constructor met message staat hieronder al. de message typ je dan op moment dat je de recordnotfound gebruik in service